test/test_smbo/test_epils.py

```diff
@@ -0,0 +1,128 @@
+
+    def get_array(self):
+        return [self.value]
+
+
+class TestSMBO(unittest.TestCase):
+
+    def setUp(self):
+        self.scenario = Scenario({'cs': test_helpers.get_branin_config_space(),
+                                  'run_obj': 'quality',
+                                  'output_dir': '',
+                                  'runcount_limit':1,
+                                  'deterministic': True})
+
+    def branin(self, config):
+        print(config)
+        y = (config[1] - (5.1 / (4 * np.pi ** 2)) * config[0] ** 2 + 5 * config[0] / np.pi - 6) ** 2
+        y += 10 * (1 - 1 / (8 * np.pi)) * np.cos(config[0]) + 10
+
+        return y
+    
+    def test_epils(self):
+        taf = ExecuteTAFuncArray(ta=self.branin)
+        epils = EPILS(self.scenario, tae_runner=taf)
+        inc = epils.optimize()
+        # not enough runs available to change the inc
+        self.assertEqual(inc["x"], 2.5)
+        self.assertEqual(inc["y"], 7.5)
+
+
+    def test_init_only_scenario_runtime(self):
+        self.scenario.run_obj = 'runtime'
+        self.scenario.cutoff = 300
+        epils = EPILS(self.scenario).solver
+        self.assertIsInstance(epils.model, RandomForestWithInstances)
+        self.assertIsInstance(epils.rh2EPM, RunHistory2EPM4LogCost)
+        self.assertIsInstance(epils.acquisition_func, LogEI)
+
+    def test_init_only_scenario_quality(self):
+        epils = EPILS(self.scenario).solver
+        self.assertIsInstance(epils.model, RandomForestWithInstances)
+        self.assertIsInstance(epils.rh2EPM, RunHistory2EPM4Cost)
+        self.assertIsInstance(epils.acquisition_func, EI)
+
+    def test_init_EIPS_as_arguments(self):
+        for objective in ['runtime', 'quality']:
+            self.scenario.run_obj = objective
+            types, bounds = get_types(self.scenario.cs, None)
+            umrfwi = UncorrelatedMultiObjectiveRandomForestWithInstances(
+                ['cost', 'runtime'], types, bounds)
+            eips = EIPS(umrfwi)
+            rh2EPM = RunHistory2EPM4EIPS(self.scenario, 2)
+            epils = EPILS(self.scenario, model=umrfwi, acquisition_function=eips,
+                        runhistory2epm=rh2EPM).solver
+            self.assertIs(umrfwi, epils.model)
+            self.assertIs(eips, epils.acquisition_func)
+            self.assertIs(rh2EPM, epils.rh2EPM)
+
+    def test_rng(self):
+        epils = EPILS(self.scenario, rng=None).solver
+        self.assertIsInstance(epils.rng, np.random.RandomState)
+        self.assertIsInstance(epils.num_run, int)
+        epils = EPILS(self.scenario, rng=1).solver
+        rng = np.random.RandomState(1)
+        self.assertEqual(epils.num_run, 1)
+        self.assertIsInstance(epils.rng, np.random.RandomState)
+        epils = EPILS(self.scenario, rng=rng).solver
+        self.assertIsInstance(epils.num_run, int)
+        self.assertIs(epils.rng, rng)
+        # ML: I don't understand the following line and it throws an error
```

Comment:
If this comment is still true the unittests should fail. As they don't I assume this issue is solved and the comment can be removed.